README.md

```diff
@@ -52,184 +52,75 @@ configuration are:
+  %%                                (%%%%%%%%%%%%%%%%%%%  *****  **
+    %%%%%        %%%%%%%%%%%%%%%
+    %%%%%%%%%%                     %%          **             ***
+       %%%                         %%  %%             %%%           %%%%,
+       %%%      %%%   %%%   %%%%%  %%%   %%%   %%  %%%   %%%  %%%       (%%
+       %%%      %%%   %%%  %%%     %%     %%/  %%  %%%   %%%  %%%  %%%%%%%%
+       %%%      %%%   %%%  %%%     %%%   %%%   %%  %%%   %%%  %%% %%%   %%%
+       %%%        %%%%%    %%%       %%%%%     %%  %%%    %%  %%%   %%%%%
+
+  This command-line tool interacts with Turbinia's API server.
+
+  You can specify the API server location in ~/.turbinia_api_config.json
+
+Options:
+  -c, --config_instance TEXT  A Turbinia instance configuration name.
+                              [default: (dynamic)]
+  -p, --config_path TEXT      Path to the .turbinia_api_config.json file..
+                              [default: (dynamic)]
+  -h, --help                  Show this message and exit.
 
 Commands:
-  <command>
-    config              Print out config file
-    testnotify          Sends test notification
-    rawdisk             Process RawDisk as Evidence
-    googleclouddisk     Process Google Cloud Persistent Disk as Evidence
-    googleclouddiskembedded
-                        Process Google Cloud Persistent Disk with an embedded
-                        raw disk image as Evidence
-    rawmemory           Process RawMemory as Evidence
-    directory           Process a directory as Evidence
-    compresseddirectory
-                        Process a compressed tar file as Evidence
-    hindsight           Process ChromiumProfile as Evidence
-    listjobs            List all available Jobs. These Job names can be used by
-                        --jobs_allowlist and --jobs_denylist
-    psqworker           Run PSQ worker
-    celeryworker        Run Celery worker
-    status              Get Turbinia Task status
-    gcplogs             Collects Turbinia logs from Stackdriver.
-    dumpgcs             Get Turbinia results from Google Cloud Storage.
-    server              Run Turbinia Server
+  config    Get Turbinia configuration.
+  evidence  Get or upload Turbinia evidence.
+  jobs      Get a list of enabled Turbinia jobs.
+  result    Get Turbinia request or task results.
+  status    Get Turbinia request or task status.
+  submit    Submit new requests to the Turbinia API server.
 ```
 
-The commands for processing evidence specify the metadata about that evidence
-for Turbinia to process. By default, when adding new evidence to be processed,
-turbiniactl will act as a client and send a request to the configured Turbinia
-server, otherwise if `server` is specified, it will start up its own Turbinia
-server process. Here's the turbiniactl usage for adding a raw disk type of
-evidence to be processed by Turbinia:
+Check out the `turbinia-client` documentation page for a detailed user guide.
```

Comment:
Do you want to add a link to this page here?

Reply:
Done